Documentation-Requirements.md

```diff
@@ -177,3 +195,41 @@ source file and re-generating the binary format image file.
 Ideally, the format of all image source files is an open standard, non-binary
 one such as SVG. Text formats are highly preferable because you can manipulate
 and compare them with standard tools (e.g. `diff(1)`).
+
+# Spelling
+
+Since this project uses a number of terms not found in conventional
+dictionaries, we have a
+[spell checking tool](https://github.com/kata-containers/tests/tree/master/cmd/check-spelling)
+tool that checks both dictionary words and the additional terms we use.
+
+Run the spell checking tool on your document before raising a PR to ensure it
+is free of mistakes.
+
+If your document introduces new terms, the custom dictionary used by the spell
+checking tool will need to be updated to incorporate the new words.
```

Comment:
Lines 209-2010 suggested rewrite: 
If your document introduces new terms, you need to update the custom dictionary used by the spell checking tool to incorporate the new words.